pinot-controller/src/main/java/org/apache/pinot/controller/ControllerStarter.java

```diff
@@ -273,8 +273,14 @@ private void setUpPinotController() {
         new SegmentCompletionManager(helixParticipantManager, _pinotLLCRealtimeSegmentManager, _controllerMetrics,
             _controllerLeadershipManager, _config.getSegmentCommitTimeoutSeconds());
 
-    _realtimeSegmentsManager = new PinotRealtimeSegmentManager(_helixResourceManager, _controllerLeadershipManager);
-    _realtimeSegmentsManager.start(_controllerMetrics);
+    if (_config.getHLCTablesAllowed()) {
+      LOGGER.info("HLC table is allowed. HLC realtime segment completion is enabled.");
```

Comment:
I suggest the log message to be "Realtime tables with High Level consumers will be supported"

Reply:
Updated.